Python/getargs.c

```diff
@@ -1979,10 +1980,19 @@ parser_init(struct _PyArg_Parser *parser)
     const char * const *keywords = parser->keywords;
     assert(keywords != NULL);
 
-    if (parser->initialized) {
+    if (_Py_atomic_size_get(&parser->initialized)) {
         assert(parser->kwtuple != NULL);
         return 1;
     }
+    PyThread_acquire_lock(_PyRuntime.getargs.mutex, WAIT_LOCK);
```

Comment:
FWIW, the function is a bit cleaner and less prone to accidentally missing a case if you factor out an inner function like I did in gh-95910.

Reply:
done